### PR TITLE
The Roadies Hammer (Or how I learnt to stop being "subtle" and write it in the code)

### PR DESCRIPTION
**Pull request by AsterixCodix**
## About The Pull Request

Makes it abundantly clear, in bright red text when Roadies spawn, that they are not allowed to use the shit in the shop to do shit in shitting bunkers for shits and gigs. Yes I just said shit 5 times, deal with it.

## Why It's Good For The Game

Saves me from bwoinking every Roadie saying "that shit ain't Kosher". However, jobbans will be issued if it still gets ignored, so enjoy that shit.

The "shit" counter is up to 8.

## Changelog
:cl:
tweak: Roadie Description
/:cl:

### CODE REVIEW

**Comment by Pebblesthe2nd** [comment]
Wouldn't it make sense to ID lock them out of the shop if they aren't supposed to be in there?
Or are they like shopkeepers?

File: code/modules/jobs/job_types/vtcc.dm [hunk 1]
```diff
@@ -622,6 +622,7 @@
 	spawn_positions = 1
 	supervisors = "the Merchant & the Shopkeeper."
 	description = "Operate the shop when the Merchant's asleep or just isn't there, push and pull those crates to and fro like your life depends on it (and when your paycheck is based on performance, well, it certainly does). If there are no crates to pick up, head out into the wastes and start scavenging for anything you could sell at an extortionate price."
```
Comment (buffyuwu) [inline code/modules/jobs/job_types/vtcc.dm:624]: this kind of suggests they go scav boss hunting despite your new forbids text though, may want to clarify it like 

``` suggestion
	description = "Operate the shop when the Merchant's asleep or just isn't there, push and pull those crates to and fro like your life depends on it (and when your paycheck is based on performance, well, it certainly does). If there are no crates to pick up, head out into the wastes and start scavenging for anything you could sell at an extortionate price. Just be sure to leave the stock alone. Shop inventory is for sale ONLY, and should not be taken for personal use."
```

Comment (AsterixCodix) [reply code/modules/jobs/job_types/vtcc.dm:624]: Fair point!

**Comment by panzer1944** [comment]
> Wouldn't it make sense to ID lock them out of the shop if they aren't supposed to be in there? Or are they like shopkeepers?

they're ID locked out of the backroom where the main high-end stuff the shop gets is already, but not out of the front where some lower-end stuff is. from what i've seen though they've been ignoring this, breaking the IDlock doors and just taking the good shit anyways to run off into bunkers and die immediately.

**Comment by buffyuwu** [comment]
> > Wouldn't it make sense to ID lock them out of the shop if they aren't supposed to be in there? Or are they like shopkeepers?
> 
> they're ID locked out of the backroom where the main high-end stuff the shop gets is already, but not out of the front where some lower-end stuff is. from what i've seen though they've been ignoring this, breaking the IDlock doors and just taking the good shit anyways to run off into bunkers and die immediately.

image

**Pull request merged**